### PR TITLE
Switch work private

### PR DESCRIPTION
**Pull request by Blize**
Add feature to be able to switch between private projects and work projects

### CODE REVIEW

File: src/views/AboutView.vue [hunk 1]
```diff
@@ -43,16 +43,38 @@
...
-					<div class="flip-card-front">
-						<h2>{{ project.projectTitle }}</h2>
-						<h4>{{ project.projectInfo }}</h4>
+			<div class="switch-work-private">
+				<h4 @click="changeProject('private-project', 'work-project', true)" id="private-project">Private</h4>
+				<h4 @click="changeProject('private-project', 'work-project', false)" id="work-project">Work</h4>
+			</div>
+
+			<div v-if="this.switchProject" class="project-wrapper">
+				<div class="flip-card" v-for="project in projects" :key="project.projectTitle">
```
Comment (slowlydev) [inline src/views/AboutView.vue:52]: Why not use a computed projects where u either return workProjects or projects, so u don't have to write the html twice. :)

File: src/views/AboutView.vue [hunk 2]
```diff
@@ -139,6 +181,25 @@ export default {
 				}
 			})
 		},
+		changeProject(id, secondId, bool) {
+			let element = document.getElementById(id)
```
Comment (slowlydev) [inline src/views/AboutView.vue:185]: You shouldn't use getElementById when using a Framework use refs instead.

File: src/views/AboutView.vue [hunk 3]
```diff
@@ -238,7 +320,7 @@ export default {
 }
 .flip-card {
 	background-color: transparent;
-	width: 300px;
+	width: 325px;
```
Comment (slowlydev) [inline src/views/AboutView.vue:323]: great choice of 325px

**Review by slowlydev — CHANGES_REQUESTED** [review]

File: src/App.vue [hunk 4]
```diff
@@ -24,7 +24,7 @@ export default defineComponent({
 		Footer,
 	},
 	mounted() {
-		var theme = localStorage.getItem('myTheme')
```
Comment (simylein) [inline src/App.vue:27]: why is it let? if it is not changed you should consider using const

File: src/views/AboutView.vue (hunk 1 above)
Comment (Blize) [reply src/views/AboutView.vue:52]: thx

File: src/App.vue [hunk 5]
```diff
@@ -56,10 +56,19 @@ body {
 
 	--light-background: white;
 	--light-secondary-background: whitesmoke;
+	--light-selected-color: rgb(212, 212, 212);
+	--light-hover-color: whitesmoke;
```
Comment (simylein) [inline src/App.vue:60]: how about always using the same method to declare colors? how about settling to hex?

File: src/App.vue [hunk 6]
```diff
@@ -56,10 +56,19 @@ body {
...
 	--light-border-color: rgb(104, 104, 104);
 
-	--shadow: box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.1);
+	--shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.1);
+}
+
+.light-theme .switch-work-private h4:hover {
+	background-color: var(--light-hover-color);
+}
+.dark-theme .switch-work-private h4:hover {
```
Comment (simylein) [inline src/App.vue:70]: this is a terrible way to do white-mode and dark-mode. please do with swapping variables

File: src/views/AboutView.vue [hunk 7]
```diff
@@ -107,6 +130,25 @@ export default {
 					projectURL: 'https://github.com/Blize/Portfolio',
 				},
 			],
+			workProjects: [
+				{
+					projectTitle: 'Onboarding',
```
Comment (slowlydev) [inline src/views/AboutView.vue:135]: why write project every time, I suggest changing it to title, info, leader

**Review by slowlydev — CHANGES_REQUESTED** [review]

**Review by simylein — CHANGES_REQUESTED** [review]
This is kinda spaghetti code

File: src/views/AboutView.vue (hunk 3 above)
Comment (Blize) [reply src/views/AboutView.vue:323]: Idk was testing something

**Comment by slowlydev** [comment]
Screenshot 2022-11-24 at 11 21 24
I don't know which projects are selected the first time

**Pull request merged**